src/zelos/config_gen.py

```diff
@@ -227,3 +233,18 @@ def generate_config_from_cmdline(cmdline_string):
     config = parser.parse_args(cmdline_string)
 
     return config
+
+
+class _ParseEnvVars(argparse._AppendAction):
+    def __call__(self, parser, namespace, arg, option_string=None):
+        d = {}
+
+        if arg:
```

Comment:
*nitpick* I think we should prefer more descriptive conditions for `if` so that someone unfamiliar to the code (myself or Kevin) may easily understand what is going on.

Reply:
Updated to exclude strings of whitespace, and makes it clear that `arg` is the user provided string that we're parsing.